harvester/create.go

```diff
@@ -25,6 +26,7 @@ const (
 	diskNamePrefix      = "disk"
 	interfaceNamePrefix = "nic"
 	poolNameLabelKey    = "harvesterhci.io/machineSetName"
+	clusterNameLabelKey = "cluster.kubernetes.io/name"
```

Comment:
Can we redefine this with `harvesterhci.io` domain? I'm not sure whether using `kubernetes.io` will influence other components someday.